platforms/jvm/language-java/src/main/java/org/gradle/api/internal/tasks/compile/incremental/recomp/AbstractRecompilationSpecProvider.java

```diff
@@ -119,7 +119,7 @@ private void processSourceChanges(CurrentCompilation current, SourceFileChangePr
         }
     }
 
-    private String rebuildClauseForChangedNonSourceFile(FileChange fileChange) {
+    private static String rebuildClauseForChangedNonSourceFile(FileChange fileChange) {
```

Comment:
What's the reason behind this change?

Reply:
This is a utility method so is a fine candidate to be made static. This is in line with other similar methods within this class